presto-docs/src/main/sphinx/release/release-0.284.rst

```diff
@@ -0,0 +1,78 @@
+* Adds `noisy_avg_gaussian(col, noiseScale[, lower, upper][, randomSeed])` aggregation which calculates the average (arithmetic mean) of all over the input values, and then adds random Gaussian noise with 0 mean and standard deviation of ``noise_scale`` to the true avg. This also provides options to clip values to a range `[lower, upper]` and a random seed for reproducibility. All values are converted to `double` before being added to the sum, which which is used to compute the avg, and the return type is `double`. When there are no input rows, this function returns ``NULL``. When generating noise, if randomSeed is omitted, SecureRandom is used; otherwise, Random is used.
+* Change InternalResourceGroupManager refresh executor's thread name for easier read.
+* Disable the optimizer PullUpExpressionInLambdaRules which can lead to query failures. The failure happens when a "CASE" expression is in a lambda expression and not using the lambda arguments. The error message will be "Can not compile special form: WHEN".
+* Enhance join-reordering to work with non-simple equi-join predicates.
+* MinN/maxN's second argument is now checked to be unique.
+* Node config property `node.ip` is now a legacy config. It is replaced with `node.internal-address` to support hostname/FQDN too.
+* Optimize  `SqlStageExecution.checkAllTaskFinal` call.
+* Reduce dns lookup overhead on NodeScheduler. This can avoid unnecessary dns lookup for HostAddress has port, while previous would incurred additional overhead when dns too slow on some host environment.
+* Reduce number of call to  `SqlStageExecution.checkAllTaskFinal`.
+* Skip access control checks for IS NULL and IS NOT NULL.
+* The Iceberg connector now supports `TRUNCATE TABLE <table>` statements.
+* This commit adds `noisy_count_if_gaussian(condition, noiseScale[, randomSeed])` aggregation which calculates the number of `TRUE` input values, and then adds random Gaussian noise with 0 mean and standard deviation of ``noise_scale`` to the true count. Optional randomSeed is used to get a fixed value of noise, often for reproducibility purposes. If randomSeed is omitted, SecureRandom is used. If randomSeed is provided, Random is used.
+
+Iceberg Connector Changes
+_________________________
+* Enforce copy-on-write mode for Iceberg tables. This can disabled with the ``merge_on_read_enabled`` session property or the ``iceberg.enable-merge-on-read-mode``configuration property.
```

Comment:
```suggestion
* Enforce copy-on-write mode for Iceberg tables. This can be disabled with the ``merge_on_read_enabled`` session property or the ``iceberg.enable-merge-on-read-mode`` configuration property.
```